.github/workflows/test.yaml

```diff
@@ -5,7 +5,8 @@ on:
 
 jobs:
   unit-tests:
-    uses: canonical/operator-workflows/.github/workflows/test.yaml@main
+    uses: canonical/operator-workflows/.github/workflows/test.yaml@remove_default_jammy_label
```

Comment:
Just to be sure: is expected to use/merge using this branch, right?

Reply:
Also because I'm not sure how to best proceed in this situation, maybe we can do a more complex logic and check if the labels has a release different than jammy, use that. Otherwise set jammy by default.

It's still unclear to me so I think keeping it like this for this PR is fine for now

Reply:
@Thanhphan1147  You might want to create a PR on operator workflows and we can discuss this there. It probably makes sense to remove the hardcoded jammy label and replace it with something like `inputs.self-hosted-runner-image` with a default of jammy.

Reply:
Got it, I will create a PR for operator workflows and let's discuss there.

Reply:
PR opened : https://github.com/canonical/operator-workflows/pull/345